### ISSUE TITLE
Implement support for "CIECAM02 JMh to XYZ" conversion.

### ISSUE THREAD

**Issue by crowsonkb** [colour-science/colour#310]
The inverse transform in colour.models.ucs_luo2006 outputs JMh and so I am not able to convert back from the new UCS spaces, lacking any way to recover JCh from JMh.

**Comment by KelSolaar** [comment]
@crowsonkb: Hi Katherine, you are absolutely right, it is something we have overlooked, actually not exactly because I noticed it was impossible to go back to CIE XYZ with the JCh correlates when I worked on [Colour - Unity](https://github.com/colour-science/colour-unity/tree/master/Assets/Colour/Shaders).

I didn't look how to do so precisely, three reasons for that:

- The API tends to be in a feature lock state because of some complex work related to metadata: https://github.com/colour-science/colour/issues/267, *metadata\** branches on the repo to get an idea: https://github.com/colour-science/colour/branches/all
- JCh Implies some substantial API changes that need to be discussed, e.g. should we pass the correlates passed as a consolidated array instead of separated values, how do we handle the current definition signature, etc...?
- Lack of time on my end and Michael

We can start looking at it though, dragging @MichaelMauderer to the conversation.
